### DISCUSSION TITLE
3rd monitor error issue

### DISCUSSION THREAD

**Discussion by abcbbong** [martinet101/ElevenClock#222] [General]
Thank you for elevenclock first, however, I have some error issue with it

I use my laptop and use 2 other monitor.

I use 4K as main and 1080p for 2nd and 3rd (laptop monitor).

as default setting, clocks are staying center of main screeen. 

I tried "Alternative clock alignment", but it shows half of clock and half of date, and only show clock on the other monitor.

![eleven clock error](https://user-images.githubusercontent.com/57967673/139777779-c4edca9c-729f-4368-958c-0a856078b367.jpg)

**Comment by marticliment** [comment]
Could you please send me the log?

**Reply by marticliment** [reply]
ElevenClock Settings -> Scroll to the bottom -> Open log

**Comment by abcbbong** [comment]
--------------------------------------------------------------------------------------------------- ElevenClock's v2.5 log: Select all the text and hit Ctrl+C to copy it ---------------------------------------------------------------------------------------------------    enNullColorNullColor['215,226,228', '160,174,183', '101,116,134', '81,92,107', '69,78,94', '41,47,64', '15,18,36', '239,105,80']Starting update check60.73046875<PySide2.QtGui.QScreen(0x18056bbc370, name="\\\\.\\DISPLAY2") at 0x00000180575248C0> PySide2.QtCore.QRect(0, 0, 3840, 2160) (3840, 2160, 0, 0, 192.0, 192.0)This is primay screen and is set to be skipped<PySide2.QtGui.QScreen(0x18056bbc410, name="\\\\.\\DISPLAY1") at 0x000001805CE46B00> PySide2.QtCore.QRect(998, 2160, 1920, 1080) (1920, 1080, 998, 2160, 120.0, 120.0)[WinError 259] No more data is available[WinError 259] No more data is availabletaskbar at bottomUsing win32 API positioning systemClock geometry: PySide2.QtCore.QRect(2720, 3180, 117, 38)Full screen rect: (998, 2160, 2918, 3240)<PySide2.QtGui.QScreen(0x18056bbc290, name="\\\\.\\DISPLAY5") at 0x000001805CE46C40> PySide2.QtCore.QRect(3840, -15, 1920, 1080) (1920, 1080, 3840, -15, 96.0, 96.0)[WinError 259] No more data is availabletaskbar at bottomUsing win32 API positioning systemClock geometry: PySide2.QtCore.QRect(5602, 1017, 75, 24)Full screen rect: (3840, -15, 5760, 1065)screenCheckThreadRestart loop: 1updates not foundBackground widget is bigger than parent!Background widget is bigger than parent!480NullColorNullColor['215,226,228', '160,174,183', '101,116,134', '81,92,107', '69,78,94', '41,47,64', '15,18,36', '239,105,80']  Sent from Mail for Windows From: Martí ClimentSent: Tuesday, November 2, 2021 11:38 AMTo: martinet101/ElevenClockCc: Jae Hoon Lee; AuthorSubject: Re: [martinet101/ElevenClock] 3rd monitor error issue (Discussion #222) ElevenClock Settings -> Scroll to the bottom -> Open log—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
